mifosng-provider/src/main/java/org/mifosplatform/organisation/staff/api/StaffApiResource.java

```diff
@@ -49,7 +49,7 @@
      * {@link StaffData}.
      */
     private final Set<String> RESPONSE_DATA_PARAMETERS = new HashSet<>(Arrays.asList("id", "firstname", "lastname", "displayName",
-            "officeId", "officeName", "isLoanOfficer", "externalId", "mobileNo", "allowedOffices", "isActive", "joiningDate"));
+            "officeId", "officeName", "isLoanOfficer", "externalId", "mobileNo", "allowedOffices", "isActive", "joiningDate", "forceStatus"));
```

Comment:
Force Status should be given only for request data parameters. This parameter is used only for forcing staff deletion through request. So please remove this parameter for response parameters